front/src/styles/scss/applications/stdcm/_linkedPath.scss

```diff
@@ -0,0 +1,90 @@
+.grey50 {
+  color: var(--grey50);
+}
+
+.grey80 {
+  color: var(--grey80);
+}
+
+.info60 {
+  color: var(--info60);
+}
+
+.stdcm-linked-path-search-container {
+  max-width: 450px;
+  align-self: flex-end;
+
+  &.anterior-linked-path {
+    margin-bottom: 18px;
+  }
+
+  &.posterior-linked-path {
+    margin-top: 18px;
+  }
+
+  .stdcm-card__header {
+    button {
+      font-size: 14px;
+      color: var(--primary60);
+      font-weight: 500;
+    }
+  }
+
+  .stdcm-card__body {
+    width: 450px;
+
+    &.add-linked-path,
+    &.linked-path {
+      padding: 11px 8px 13px;
+    }
+  }
+
+  .stdcm-linked-path-button {
```

Comment:
You can move the class stdcm-linked-path-button in linked-path (you can add a class .linked-path)